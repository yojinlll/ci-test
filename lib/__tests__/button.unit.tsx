import renderer from 'react-test-renderer'
import React from 'react'
import Button from '../button'

describe('button', ()=>{
  it('是个 div ', ()=>{
    const buttonJson = renderer.create(<Button/>).toJSON()
    console.log(buttonJson);
    expect(buttonJson).toMatchSnapshot()
  })
})