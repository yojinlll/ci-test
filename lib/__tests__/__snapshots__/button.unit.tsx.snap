// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button 是个 Button  1`] = `
<div>
  button
</div>
`;

exports[`button 是个 body  1`] = `
<div>
  button
</div>
`;

exports[`button 是个 button  1`] = `
<button>
  button
</button>
`;

exports[`button 是个 div  1`] = `
<div>
  button
</div>
`;
